packages/shared/src/hooks/bookmark/useMoveBookmarkToFolder.ts

```diff
@@ -0,0 +1,29 @@
+import { useMutation } from '@tanstack/react-query';
+import { moveBookmarkToFolder } from '../../graphql/bookmarks';
+import { EmptyResponse } from '../../graphql/emptyResponse';
+import { useLogContext } from '../../contexts/LogContext';
+import { LogEvent } from '../../lib/log';
+
+interface UseMoveBookmarkToFolder {
+  isPending: boolean;
+  moveBookmarkToFolder: (
+    options: Record<'postId' | 'listId', string>,
+  ) => Promise<EmptyResponse>;
+}
+
+export const useMoveBookmarkToFolder = (): UseMoveBookmarkToFolder => {
+  const { logEvent } = useLogContext();
+  const { isPending, mutateAsync } = useMutation({
+    mutationFn: moveBookmarkToFolder,
+    onSuccess: () => {
+      logEvent({
+        event_name: LogEvent.MoveBookmarkToFolder,
+      });
```

Comment:
This should be a post event (postLogEvent)

Reply:
Updated ✅